sh_scrapy/utils.py

```diff
@@ -0,0 +1,6 @@
+from sh_scrapy.settings import populate_settings
+from sh_scrapy.crawl import _get_apisettings
+
+
+def get_project_settings():
+    return populate_settings(_get_apisettings())
```

Comment:
I wonder if we should also allow and propagate  `spider: bool` (or `for_spider: bool`) param to `populate_settings` call: when it's provided, it also [merges](https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy/blob/master/sh_scrapy/settings.py#L185-L186) spider settings to the final dict, it might be beneficial and important when running under ScrapyCloud, wdyt?

Reply:
I see your point, but I wonder what real-life use case could benefit from it.

The point of `get_project_settings` for me is to give a script, not a spider, access to project settings. Spiders can already access their projects settings with their regular API.

Reply:
That's fair enough, thanks for explanation, merging 👌🏻